config/jobs/kubernetes/test-infra/janitors.yaml

```diff
@@ -3,17 +3,12 @@ periodics:
   name: maintenance-ci-aws-janitor
   labels:
     preset-service-account: "true"
+  decorate: true
   spec:
     containers:
-    - args:
-      - --bare
-      - --timeout=60
-      - --service-account=/etc/service-account/service-account.json
-      - --upload=gs://kubernetes-jenkins/logs
-      - --scenario=execute
-      - --
-      - /aws-janitor
-      - --
+    - command:
+      - /app/boskos/aws-janitor/app.binary
```

Comment:
oops, this wasn't intentional. Should we update it so that it uses the previous binary name/location?

I'm also wondering if the command should actually be:
```
- "runner"
- /app/boskos/aws-janitor/app.binary
```
similar to how we generally invoke jobs based on the kubekins image (runner.sh vs runner as is configured in the base image we are using here).

Reply:
I'm not that familiar with the bazel rules to determine how we can update the binary location.

I just the command to use `runner` but now that I'm taking a closer look at the image, i think the runner script doesnt have execute permissions and its also not in the PATH so it would need to be `/runner`

```
root@111a8f99b776:/# ls -la /runner
-rw-r----- 1 root root 833 Sep 29  2018 /runner
root@111a8f99b776:/# cat runner
#!/usr/bin/env bash
# Copyright 2016 The Kubernetes Authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

set -o errexit
set -o nounset
set -o pipefail

git clone https://github.com/kubernetes/test-infra
./test-infra/jenkins/bootstrap.py \
    --job=${JOB_NAME} \
    --service-account=${GOOGLE_APPLICATION_CREDENTIALS} \
    "$@"
```

Reply:
I don't think you want to run bootstrap in a decorated job, it's just going to be doing nothing -- run the actual thing that bootstrap executes once it is done with setup

Reply:
ah ok, I've updated it to not use bootstrap